ci/Jenkinsfile

```diff
@@ -1,21 +1,17 @@
+// Pipeline for PR's, master merges and production
+
 import org.jenkinsci.plugins.pipeline.modeldefinition.Utils
 
+// Function duplicated in `Jenkinsfile.release`
 def node_info() {
     echo "Running stage on ${env.NODE_NAME}"
 }
 
-def wait_for_db() {
-    retry(6) {
-        sleep(time: 5, unit: 'SECONDS')
-        sh '/usr/bin/pg_isready --host=db --username=ci --dbname=ci --timeout=60'
-    }
-}
-
-def wait_for_pr_db() {
+// Function duplicated in `Jenkinsfile.release` without the command parameter
+def wait_for_db(String command) {
```

Comment:
Could be renamed later to match what it actually does, e.g. wait_for_command_to_succeed or similar.